runtime/stream.h

```diff
@@ -216,6 +217,6 @@ strmGetCurrFileNum(strm_t *pStrm) {
 /* prototypes */
 PROTOTYPEObjClassInit(strm);
 rsRetVal strmMultiFileSeek(strm_t *pThis, int fileNum, off64_t offs, off64_t *bytesDel);
-rsRetVal strmReadMultiLine(strm_t *pThis, cstr_t **ppCStr, regex_t *preg, sbool bEscapeLF);
+rsRetVal strmReadMultiLine(strm_t *pThis, cstr_t **ppCStr, regex_t *preg, sbool bEscapeLF, off64_t *offset);
```

Comment:
I don't know what the policy in rsyslog for support of plugins not maintained in the source tree.  Is there an rsyslog development release type packaging out there?  Or are folks just compiling against the checked-out git tree?

Certainly it would be easy to write such a compatibility layer to do that if we have to support external plugins.